temporal-test-server/src/main/java/io/temporal/internal/testservice/StateMachines.java

```diff
@@ -712,7 +703,7 @@ private static void scheduleNexusOperation(
                             .setService(attr.getService())
                             .setOperation(attr.getOperation())
                             .setPayload(attr.getInput())
-                            .setCallback(""))); // TODO(pj): support async operations
+                            .setCallback(ref.toBytes().toString())));
```

Comment:
Can you please make this a URL and put data in the callback header? This is user visible and should look roughly like what we have in the real server.

In the Go test framework (which operates on a single workflow), I used this:

```go
					Callback: "http://test-env/operations",
					CallbackHeader: map[string]string{
						// The test client uses this to call resolveNexusOperation.
						"operation-sequence": strconv.FormatInt(h.seq, 10),
					},
```